ui/src/project.vue

```diff
@@ -325,15 +323,63 @@
                             </div>
                         </div>
 
-                        <!--
-                        <div v-if="detailTab == 4">
-                            <vue-disqus ref="disqus" shortname="brain-life" :identifier="selected._id"/>
-                        </div>
-                        -->
 
+                        <div v-if="detailTab == 4">
+                            <!--<vue-disqus ref="disqus" shortname="brain-life" :identifier="selected._id"/> -->
+                            <b-alert show variant="secondary" v-if="!config.user">
```

Comment:
Should we allow comments to be exposed publicly for public project?

Reply:
This can let other people to spam